Makefile

```diff
@@ -123,14 +123,23 @@ $(DOCS_SYNOPISIS_DIR)/*.md: vendor $(ADDON_ASSET_FILE)
 .PHONY: synopsis_docs
 synopsis_docs: $(DOCS_SYNOPISIS_DIR)/*.md
 
-.PHONY: release
-release: clean fmtcheck test prerelease $(GOPATH)/bin/gh-release cross
+.PHONY: do_release
+do_release: clean fmtcheck test prerelease $(GOPATH)/bin/gh-release cross
 	mkdir -p release
 	gnutar -zcf release/minishift-$(MINISHIFT_VERSION)-darwin-amd64.tgz LICENSE README.adoc -C $(BUILD_DIR)/darwin-amd64 minishift
 	gnutar -zcf release/minishift-$(MINISHIFT_VERSION)-linux-amd64.tgz LICENSE README.adoc -C $(BUILD_DIR)/linux-amd64 minishift
 	zip -j release/minishift-$(MINISHIFT_VERSION)-windows-amd64.zip LICENSE README.adoc $(BUILD_DIR)/windows-amd64/minishift.exe
 	gh-release checksums sha256
-	gh-release create minishift/minishift $(MINISHIFT_VERSION) master v$(MINISHIFT_VERSION)
+	# TODO: Change to minishift/minishift once final review is done
+	gh-release create minishift-bot/minishift $(MINISHIFT_VERSION) master v$(MINISHIFT_VERSION)
+
+.PHONY: ci_release
+ci_release:
+	curl -s -H "$(shell curl -s --user 'minishift:$(API_KEY)' 'https://ci.centos.org//crumbIssuer/api/xml?xpath=concat(//crumbRequestField,":",//crumb)')" \
```

Comment:
That said, didn't we say that we don't beed the MILESTONE_ID? It can be determined by the RELEASE_VERSION using the GitHub API. This way we avoid mistakes where one accidentally specified a non matching RELEASE_VERSION and MILESTONE_ID.